docs/core/tools/global-tools.md

```diff
@@ -1,135 +1,288 @@
 
-## Use the tool
+```dotnetcli
+dotnet new tool-manifest
+```
+
+This command creates a manifest file named *dotnet-tools.json* under the *.config* directory. To add a local tool to the manifest file, use the [dotnet tool install](dotnet-tool-install.md) command without either the `--global` or `--tool-path` option, as shown in the following example:
 
-Once the tool is installed, you can call it by using its command. Note that the command may not be the same as the package name.
+```dotnetcli
+dotnet tool install dotnetsay
+```
+
+The command output shows which manifest file the newly installed tool is in, as shown in the following example:
+
+```
+You can invoke the tool from this directory using the following command:
+dotnet tool run dotnetsay
+Tool 'dotnetsay' (version '2.1.4') was successfully installed.
+Entry is added to the manifest file /Users/name/tools/sub/dotnet-tools.json
+```
+
+The following example shows a manifest file with two local tools installed:
+
+```json
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "botsay": {
+      "version": "1.0.0",
+      "commands": [
+        "botsay"
+      ]
+    },
+    "dotnetsay": {
+      "version": "2.1.3",
+      "commands": [
+        "dotnetsay"
+      ]
+    }
+  }
+}
+```
+
+You typically add a local tool to the root directory of the repository. After you check in the manifest file to the repository, developers who check out code from the repository get the latest manifest file. To install all of the tools listed in the manifest file, they run the `dotnet tool restore` command:
+
+```dotnetcli
+dotnet tool restore
+```
 
-If the command is `dotnetsay`, you call it with:
+The output indicates which tools were restored:
+
+```console
+Tool 'botsay' (version '1.0.0') was restored. Available commands: botsay
+Tool 'dotnetsay' (version '2.1.3') was restored. Available commands: dotnetsay
+Restore was successful.
+```
+
+## Install a specific tool version
+
+To install a pre-release version or a specific version of a tool, specify the version number by using the `--version` option, as shown in the following example:
+
+```dotnetcli
+dotnet tool install dotnetsay --version 2.1.3
+```
+
+## Use a tool
+
+The command that you use to invoke a tool may be different from the name of the package that you install. To display all of the tools currently installed on the machine for the current user, use the [dotnet tool list](dotnet-tool-list.md) command:
+
+```dotnetcli
+dotnet tool list
+```
+
+The output shows each tool's version and command, similar to the following example:
+
+```console
+Package Id      Version      Commands       Manifest
+------------------------------------------------------------------------------------------------------
+botsay          1.0.0        botsay         C:\Users\username\Documents\botsay\.config\dotnet-tools.json
+dotnetsay       2.1.3        dotnetsay      C:\Users\username\Documents\botsay\.config\dotnet-tools.json
```

Comment:
It seems odd that the manifest files is buried in the botsay directory because it's probably not a parent directory of where dotnetsay would be run from?

Reply:
Where you run a command from is determined by where you install it.  Since in this tutorial you install dotnetsay in botsay, that's where you run it from.  I could expand the scenario by installing dotnetsay in a different directory.  Do you think that is required here to make clear how it works?

Reply:
I think it's a little weird that the dotnetsay tool lives in the botsay folder. Is there a way to install them in a "Tools" folder? That would seem to make more sense to me.

Reply:
The tools folder installation sounds more appropriate to a global tool in custom location, aka tool-path tool. Local tools are tools you want to use with one project so you can do things like use a different version of the same tool with different projects.

Reply:
I agree. It's confusing to have the folder named botsay (the name of an existing tool). I would use something like `project` or `code` or `repository`

Reply:
How about this:  start the first tutorial by creating a "repository" folder, and create botsay under that, then in the local tools tutorial install the tools in the repository folder.